docker/Dockerfile.release

```diff
@@ -56,15 +56,30 @@ RUN pip3 install -U pip
 WORKDIR /opt
 
 # CMake
-RUN CMAKE_VERSION=3.17 && \
-    CMAKE_BUILD=3.17.4 && \
+RUN CMAKE_VERSION=3.18 && \
+    CMAKE_BUILD=3.18.6 && \
     wget -nv https://cmake.org/files/v${CMAKE_VERSION}/cmake-${CMAKE_BUILD}.tar.gz && \
     tar -xf cmake-${CMAKE_BUILD}.tar.gz && \
     cd cmake-${CMAKE_BUILD} && \
     ./bootstrap --parallel=$(grep ^processor /proc/cpuinfo | wc -l) -- -DCMAKE_USE_OPENSSL=OFF && \
     make -j"$(grep ^processor /proc/cpuinfo | wc -l)" install && \
     rm -rf /cmake-${CMAKE_BUILD}
 
+# Miniconda
+ENV PATH="/opt/conda/bin:${PATH}"
+RUN MINICONDA_URL="https://repo.anaconda.com/miniconda/Miniconda3-py38_4.12.0-Linux-x86_64.sh"; \
+    SHA256SUM="3190da6626f86eee8abf1b2fd7a5af492994eb2667357ee4243975cdbb175d7a"; \
+    wget "${MINICONDA_URL}" -O miniconda.sh -q && \
+    echo "${SHA256SUM} miniconda.sh" > shasum && \
```

Comment:
Wouldn't you want something like ``sha256sum -c ./shasum && \`` here to really check sha before running it?

Reply:
Good point, added.